submissions/EmmanuelIdiaghe/php/loopNum.php

```diff
@@ -0,0 +1,14 @@
+<?php
+#Using Loops
+   for ($i=100; $i <= 150; $i++) { 
+       echo $i . "<br>";
+   }
+
+   echo "#########################<br>";
+
+   for ($j=0; $j <= 50; $j++) { 
```

Comment:
Let your count start from `2` to reduce the number of iteration. It's not a must it should start from 0